cmd/receptor-cl/receptor.go

```diff
@@ -19,9 +19,10 @@ import (
 )
 
 type nodeCfg struct {
-	ID            string                       `description:"Node ID. Defaults to local hostname." barevalue:"yes"`
-	DataDir       string                       `description:"Directory in which to store node data"`
-	FirewallRules []netceptor.FirewallRuleData `description:"Firewall Rules (see documentation for syntax)"`
+	ID                       string                       `description:"Node ID. Defaults to local hostname." barevalue:"yes"`
+	DataDir                  string                       `description:"Directory in which to store node data"`
+	FirewallRules            []netceptor.FirewallRuleData `description:"Firewall Rules (see documentation for syntax)"`
+	MaxIdleConnectionTimeout string                       `description:"User defined maximum time a backend connection can go without data before we consider it failed."`
```

Comment:
We may change this to,

"Max duration with no traffic before a backend connection is timed out and refreshed."

To emphasize that the backend will timeout, but then refresh automatically.

Reply:
done :-)